samples/SmokeTest/src/main/java/com/azure/CosmosDB.java

```diff
@@ -0,0 +1,127 @@
+package com.azure;
+
+import com.azure.data.cosmos.CosmosClient;
+import com.azure.data.cosmos.CosmosContainer;
+import com.azure.data.cosmos.CosmosItemProperties;
+import com.azure.data.cosmos.FeedResponse;
+import com.azure.data.cosmos.FeedOptions;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+public class CosmosDB {
+    private static CosmosClient client;
+    private static CosmosContainer container;
+    private static final String dbName = "JavaSolarSystem";
+    private static final String collectionName = "Planets";
+
+    private static void createDatabase() {
+        System.out.print("Creating database... ");
+        client.createDatabaseIfNotExists(dbName).block();
+        System.out.println("\tDONE.");
+    }
+
+    private static void createCollection() {
```

Comment:
Using .block() changes this from being asynchronous to a synchronous call. Consider:

```java
private static Mono<CosmosContainer> createCollection(CosmosClient client) {
    return client.getDatabase(dbName).createContainer(collectionName, "/id")
        .map(response -> response.container());
}
```